device/common/include/traccc/finding/device/impl/apply_interaction.ipp

```diff
@@ -31,24 +33,29 @@ TRACCC_DEVICE inline void apply_interaction(
 
     // in param
     bound_track_parameters_collection_types::device params(params_view);
+    vecmem::device_vector<const unsigned int> params_liveness(
+        params_liveness_view);
 
     if (globalIndex >= n_params) {
         return;
     }
 
     auto& bound_param = params.at(globalIndex);
 
-    // Get surface corresponding to bound params
-    const detray::tracking_surface sf{det, bound_param.surface_link()};
-    const typename detector_t::geometry_context ctx{};
-
-    // Apply interactor
-    typename interactor_type::state interactor_state;
-    interactor_type{}.update(
-        ctx,
-        detail::correct_particle_hypothesis(cfg.ptc_hypothesis, bound_param),
-        bound_param, interactor_state,
-        static_cast<int>(detray::navigation::direction::e_forward), sf);
+    if (params_liveness.at(globalIndex) != 0u) {
```

Comment:
At the first step of CKF, I think all values of `params_liveness` is 0u, which is a default value, so this will not be passed?

Reply:
Right; to make sure that everything works as normal, this sets the values to 1 for all parameters: https://github.com/acts-project/traccc/pull/732/files#diff-90d760f48590c5507580da9993439d4bbaa1758d54e6c05d332631c9e256fbe9R280

Reply:
Ah OK I missed `memset`